source/src/drivers/cs_Storage.cpp

```diff
@@ -557,12 +557,31 @@ ret_code_t Storage::garbageCollectInternal() {
 	return fdsRetCode;
 }
 
+#if NORDIC_SDK_VERSION != 15
+/**
+ * This function can now be found as part of the example files themselves.
+ */
+static uint32_t nrf5_flash_end_addr_get()
+{
+    uint32_t const bootloader_addr = BOOTLOADER_ADDRESS;
```

Comment:
Isn't this bootloader address defined at application compile time, instead of what is actually registered?

Reply:
Ah, this isn't from our build system, but is defined in `app_util.h`.

Reply:
Fixed in https://github.com/crownstone/bluenet/pull/137/commits/8c9935e28732663fd9dff11015e4ad169cd3cc15